proto/ibc/lightclients/beefy/v1/beefy.proto

```diff
@@ -140,6 +140,9 @@ message Header {
 
   // optional payload to update the mmr root hash.
   MmrUpdateProof mmr_update_proof = 4 [(gogoproto.nullable) = true];
+
+  // previously finalized block of the relay chain
+  uint64 previously_finalized = 5;
```

Comment:
Nahhh this isn't right, clientstate already tracks this

Reply:
Oops. Forgot about that, was supposed to be experimental.